### PR TITLE
Update api doc for core/renderer/scene

### PR DESCRIPTION
**Pull request by pandamicro**
[skip ci]

### CODE REVIEW

File: cocos/core/renderer/scene/directional-light.ts [hunk 1]
```diff
@@ -218,9 +238,9 @@ export class DirectionalLight extends Light {
     }
 
     /**
-      * @en get or set shadow camera near
-      * @zh 获取或者设置阴影相机近裁剪面
-      */
+     * @en get or set shadow camera near
```
Comment (star-e) [inline cocos/core/renderer/scene/directional-light.ts]: Should it be get or set shadow camera near "clip plane"?

File: cocos/core/renderer/scene/directional-light.ts [hunk 2]
```diff
@@ -232,9 +252,9 @@ export class DirectionalLight extends Light {
     }
 
     /**
-      * @en get or set shadow camera far
-      * @zh 获取或者设置阴影相机远裁剪面
-      */
+     * @en get or set shadow camera far
```
Comment (star-e) [inline cocos/core/renderer/scene/directional-light.ts]: Should it be get or set shadow camera far "clip plane"?

File: cocos/core/renderer/scene/model.ts [hunk 3]
```diff
@@ -101,71 +101,71 @@ const lightmapSamplerWithMipHash = new SamplerInfo(
...
+     * @en Whether the model is initialized
      * @zh 是否初始化
      */
     get inited (): boolean {
         return this._inited;
     }
 
     /**
-     * @en return world aabb
+     * @en The aabb bounding box of the model in the world space
```
Comment (star-e) [inline cocos/core/renderer/scene/model.ts]: aabb means axis-aligned bounding box, so aabb "bounding box" is redundant.

File: cocos/core/renderer/scene/model.ts [hunk 4]
```diff
@@ -101,71 +101,71 @@ const lightmapSamplerWithMipHash = new SamplerInfo(
...
      * @zh 获取世界空间包围盒
      */
     get worldBounds () {
         return this._worldBounds!;
     }
 
     /**
-     * @en return model aabb
-     * @zh 获取模型包围盒
+     * @en The aabb bounding box of the model in the model space
```
Comment (star-e) [inline cocos/core/renderer/scene/model.ts]: redundant "bounding box"

File: cocos/core/renderer/scene/model.ts [hunk 5]
```diff
@@ -271,7 +271,7 @@ export class Model {
     }
 
     /**
-     * @en enabled
+     * @en Whether the model is enabled
```
Comment (star-e) [inline cocos/core/renderer/scene/model.ts]: I think the wording of "model is enabled" is inappropriate. Does it mean visible here?

File: cocos/core/renderer/scene/model.ts [hunk 6]
```diff
@@ -286,147 +286,147 @@ export class Model {
...
     /**
-     * @en model type
+     * @en The type of the model
      * @zh 模型类型
      */
     public type = ModelType.DEFAULT;
 
     /**
-     * @en model's render scene
+     * @en The render scene of the model
```
Comment (star-e) [inline cocos/core/renderer/scene/model.ts]: I think the render scene is not part of the model.
Maybe something like "The render scene where the model lives."

File: cocos/core/renderer/scene/model.ts [hunk 7]
```diff
@@ -286,147 +286,147 @@ export class Model {
...
     /**
-     * @en instance attributes
+     * @en The instance attributes
      * @zh 实例化属性
      */
     public instancedAttributes: IInstancedAttributeBlock = { buffer: null!, views: [], attributes: [] };
 
     /**
-     * @en world aabb
+     * @en The world aabb
```
Comment (star-e) [inline cocos/core/renderer/scene/model.ts]: aabb should be in uppercase, which is more common.

File: cocos/core/renderer/scene/model.ts [hunk 8]
```diff
@@ -286,147 +286,147 @@ export class Model {
...
     /**
-     * @en shadow normal bias
+     * @en Shadow normal bias
      * @zh 阴影法线偏移
      */
     protected _shadowNormalBias = 0;
 
     /**
-     * @en enable or not
+     * @en Whether the model is enable
```
Comment (star-e) [inline cocos/core/renderer/scene/model.ts]: Should be enabled

File: cocos/core/renderer/scene/model.ts [hunk 9]
```diff
@@ -641,10 +641,10 @@ export class Model {
     }
 
     /**
-     * @en create model's aabb
+     * @en Create the model's aabb bounding box
```
Comment (star-e) [inline cocos/core/renderer/scene/model.ts]: redundant "bounding box"

File: cocos/core/renderer/scene/shadows.ts [hunk 10]
```diff
@@ -260,10 +268,29 @@ export class Shadows {
     public maxReceived = 4;
 
     // local set
+    /**
+     * @internal
+     */
     public firstSetCSM = false;
+    /**
+     * @en The far clip of the shadow camera
```
Comment (star-e) [inline cocos/core/renderer/scene/shadows.ts]: I think far clip plane is better

File: cocos/core/renderer/scene/skybox.ts [hunk 11]
```diff
@@ -69,7 +69,16 @@ export const EnvironmentLightingType = Enum({
...
 });
 
+/**
+ * @en The skybox configuration of the render scene,
+ * currently some rendering options like hdr and ibl lighting configuration is also here.
+ * @zh 渲染场景的天空盒配置，目前一些渲染配置，比如 HDR 模式和环境光照配置也在 Skybox 中。
+ */
 export class Skybox {
+    /**
+     * @en The Model object for the skybox
```
Comment (star-e) [inline cocos/core/renderer/scene/skybox.ts]: Should it be  "The model object of the skybox"?

File: cocos/core/renderer/scene/sphere-light.ts [hunk 12]
```diff
@@ -43,10 +48,18 @@ export class SphereLight extends Light {
         super._destroy();
     }
 
+    /**
+     * @en The world position of the center point of the light source
```
Comment (star-e) [inline cocos/core/renderer/scene/sphere-light.ts]: I think "The world position of the light source" is simpler and better.

File: cocos/core/renderer/scene/submodel.ts [hunk 13]
```diff
@@ -133,6 +152,10 @@ export class SubModel {
         return this._subMesh!;
     }
 
+    /**
+     * @en The rendering priority of the sub model
```
Comment (star-e) [inline cocos/core/renderer/scene/submodel.ts]: "sub-model" might be better.

**Comment by star-e** [comment]
Summary:
1. `aabb`, `aabb bounding box` should be renamed to `AABB`, which is the appropriate abbreviation.
2. It should be `camera near/far clip plane`, not `camera near/far` or `camera near/far clip`.

**Review by star-e — APPROVED** [review]

File: cocos/3d/lights/spot-light-component.ts [hunk 14]
```diff
@@ -228,7 +228,7 @@ export class SpotLight extends Light {
     }
 
     /**
-     * @en get or set shadow map sampler offset
+     * @en The sampler offset of the shadow map.
```
Comment (jk20012001) [inline cocos/3d/lights/spot-light-component.ts]: The depth offset of shadow to avoid moire pattern artifacts
阴影的深度偏移, 可以减弱跨像素导致的条纹状失真

**Comment by pandamicro** [comment]
https://github.com/cocos/cocos-engine/issues/10840

**Pull request merged**